src/sentry-dotnet-unity/Sentry.Unity/link.xml

```diff
@@ -1,62 +0,0 @@
-﻿<linker>
-    <assembly fullname="Sentry" preserve="all"/>
-
-    <assembly fullname="System">

```

Comment:
We might not need this now with System.Text.Json but it's something we need to keep an eye on. On IL2CPP the linker drops anything that's not accessible statically and this is the way around that

Reply:
Didn't get that.